src/main/python/pybuilder/plugins/python/pylint_plugin.py

```diff
@@ -44,4 +45,63 @@ def execute_pylint(project, logger):
     logger.info("Executing pylint on project sources")
 
     command_and_arguments = ["pylint"] + project.get_property("pylint_options")
-    execute_tool_on_modules(project, "pylint", command_and_arguments, True)
+    pylint_output_file_path = execute_tool_on_modules(project, "pylint", command_and_arguments, True)[1]
+
+    with open(pylint_output_file_path, 'r') as f:
+        file_content = f.read().splitlines()
+
+    module_name = ''
+    pylint_score = 0
+    pylint_change = 0
+    errors = 0
+    errors_info = ''
+    warnings = 0
+    warnings_info = ''
+    show_info_messages = project.get_property('pylint_show_info_messages')
+    show_warning_messages = project.get_property('pylint_show_warning_messages')
+    for line in file_content:
+        if line.startswith('************* Module'):
+            module_name = line.split(' ')[2]
+
+        if line.startswith('E:') or line.startswith('F:'):
+            logger.error('Pylint: Module %s: ' % module_name + line)
+            errors += 1
+
+        if show_warning_messages and line.startswith('W:'):
+            logger.warn('Pylint: Module %s: ' % module_name + line)
+            warnings += 1
+
+        if show_info_messages and (line.startswith('C:') or line.startswith('R:')):
+            logger.info('Pylint: Module %s: ' % module_name + line)
```

Comment:
Hi. @maciejpolanczyk 
`logger.info` is the highest level of logger and always shown. I think that for conventions/refactorings should be used `logger.debug`